test/unit/views/CommodityCodeDigitsViewSpec.scala

```diff
@@ -28,12 +28,14 @@ class CommodityCodeDigitsViewSpec extends StringViewBehaviours {
 
   val form = new CommodityCodeDigitsFormProvider()()
 
-  def createView = () => commodityCodeDigits(frontendAppConfig, form, NormalMode)(fakeRequest, messages)
+  val goodsName = "goods"
```

Comment:
I think this should have a different name from the default so we can verify we aren't getting the default